pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -7725,6 +7725,85 @@ def test_multifile_read_check_long_list(tmp_path):
 
     return
 
+def test_multifile_read_check_ValueError(tmp_path):
```

Comment:
This looks almost the same as the KeyError test, might be a good candidate for a `pytest.mark.parametrize`?

Reply:
I'm not sure it would totally make sense, because what I'm doing to the test files to break them is different for each error type, but I can play around with trying it more if you'd like.

Reply:
It's up to you, and it's just about reducing duplicated code. You can use an if statement to things slightly differently for each case if most of the code is the same. But if they're really different it might not be worth the effort.

Reply:
I went ahead and did it - you were right that it made things more efficient.